Kernel/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_compile_options(-Os)
+add_compile_options(-Os -ggdb3)
```

Comment:
Let's not slow down everyone's kernel build with this :D

Reply:
Oops, how did that manage to slip in? :see_no_evil: